t/57-oracle-datetime.t

```diff
@@ -83,6 +89,73 @@ isa-ok $sth.column-types[0], DateTime, 'CURRENT_TIMESTAMP is DateTime';
+note '#+ -------------------------------------------------------- +';
+
+%con-parms = :database<XE>, :username<TESTUSER>, :password<Testpass>, :!AutoCommit;
+
+try {
+  $dbh = DBIish.connect('Oracle', |%con-parms);
+  CATCH {
+	    when X::DBIish::LibraryMissing | X::DBDish::ConnectionFailed {
+		diag "$_\nCan't continue.";
+	    }
+            default { .rethrow; }
+  }
+}
+
+# session mgmt comming soon
+is $dbh.do('ALTER SESSION SET time_zone               = \'-00:00\''), 0, 'ALTER SESSION ...';
+is $dbh.do('ALTER SESSION SET nls_date_format         = \'YYYY-MM-DD"T"HH24:MI:SS"Z"\''), 0, 'ALTER SESSION ...';
+is $dbh.do('ALTER SESSION SET nls_timestamp_format    = \'YYYY-MM-DD"T"HH24:MI:SS"Z"\''), 0, 'ALTER SESSION ...';
+is $dbh.do('ALTER SESSION SET nls_timestamp_tz_format = \'YYYY-MM-DD"T"HH24:MI:SS"Z"\''), 0, 'ALTER SESSION ...';
+is $dbh.execute('SELECT * FROM test_datetime').allrows, (), 'Perfect table is empty!';
+
+$sth = $dbh.prepare(
+    q|INSERT INTO test_datetime
+    FIELDS (adate, atimestamp0, atimestamp6, atimestamp0tz, atimestamp6tz, atimestamp0ltz, atimestamp6ltz)
+    VALUES (?,?,?,?,?,?,?)|);
+$now = DateTime.now.truncated-to('second').in-timezone(0);
+
+say '# Tasty timestamp: ', $now;
+lives-ok {
+    $sth.execute(
+        $now, # Need an ISO full date
+	$now, $now, $now, $now, $now, $now
+    );
+},                                           'Can insert Perl6 values';
+
+is $dbh.execute('SELECT * FROM test_datetime').allrows.elems, 1, 'One row!';
+
+# DATE
+is $dbh.execute('SELECT adate FROM test_datetime').allrows, $now.Date.Str, 'DATE is read into Date.new() but lacks the time component!';
+# TIMESTAMP(0)/(N)
+is $dbh.execute('SELECT atimestamp0 FROM test_datetime').allrows, $now.Str, 'TIMESTAMP(0) is read into DateTime.new()';
+is $dbh.execute('SELECT atimestamp6 FROM test_datetime').allrows, $now.Str, 'TIMESTAMP(6) is read into DateTime.new()';
+# TIMESTAMP(0)/(N) WITH TIME ZONE
+is $dbh.execute('SELECT atimestamp0tz FROM test_datetime').allrows, $now.Str, 'TIMESTAMP(0) W TZ is read into DateTime.new()';
+is $dbh.execute('SELECT atimestamp6tz FROM test_datetime').allrows, $now.Str, 'TIMESTAMP(6) W TZ is read into DateTime.new()';
+# TIMESTAMP(0)/(N) WITH LOCAL TIME ZONE
+is $dbh.execute('SELECT atimestamp0ltz FROM test_datetime').allrows, $now.Str, 'TIMESTAMP(0) W LTZ is read into DateTime.new()';
+is $dbh.execute('SELECT atimestamp6ltz FROM test_datetime').allrows, $now.Str, 'TIMESTAMP(6) W LTZ is read into DateTime.new()';
+
+$dbh.rollback;
+ok $sth.dispose,  'dispose';
+is $dbh.execute('SELECT * FROM test_datetime').allrows.elems, 0, 'Perfect table is empty!';
+
+note '#+ -------------------------------------------------------- +';
+note '## Issue #204 - ADDED Missing TIMESTAMP Support (END)';
+note '#+ -------------------------------------------------------- +';
+
+# Done
+ok $dbh.execute($dropper), 'Table Cleanup';
+ok $dbh.dispose, 'Say good night!';
```

Comment:
Also as above, those don't need to be tests. Might be better to move them to an END block (or something similar to a Perl 5 END block).

Reply:
No such thing as too many tests; Feel free to do whatever with them.